cvmfs/cvmfs.cc

```diff
@@ -1540,13 +1540,13 @@ static void cvmfs_getxattr(fuse_req_t req, fuse_ino_t ino, const char *name,
     download_manager_->GetTimeout(&seconds, &seconds_direct);
     attribute_value = StringifyInt(seconds_direct);
   } else if (attr == "user.rx") {
-    int64_t rx = 
+    int64_t rx =
```

Comment:
By the way, these are fixes for the whitespaces that were there from a previous push.